client/src/app/app.module.ts

```diff
@@ -16,14 +16,14 @@ import { AppRoutingModule } from './app-routing.module';
 import { OpComponent } from './op.component';
 import reducer from './reducers';
 
-import { HeaderComponent } from './components/header/header.component';
```

Comment:
Why change the imports? There shouldn't be a `inedx.ts` file in `components` directory so your `import` shouldn't work.